src/test/java/org/jenkinsci/plugins/workflow/cps/CpsVmExecutorServiceTest.java

```diff
@@ -179,7 +178,7 @@
             });
             errors.checkSucceeds(() -> {
                 p.setDefinition(new CpsFlowDefinition(
-                    "def cs = [ action: {-> sleep(1) } ]\n" +
+                    "def cs = [ action: {-> echo 'test' } ]\n" +
```

Comment:
I do not think you want to do that. `echo` is synchronous, which is why these tests use `sleep`. Anyway edited in #320.